### ISSUE TITLE
CUDA OUT OF  MEMORY

### ISSUE THREAD

**Issue by yangkang2318** [amzn/amazon-weak-ner-needle#2]
Hello,
When I run "roberta_mlm_pretrain.sh",it always occur the error that CUDA out of memory(I have 4 TITAN RTX GPUs,which means the memory is enough).Is it because the data file all_text.txt is too big？
PS:The terminal show the info:INFO - main - Creating features from dataset file at ../bio_script/tasks/unlabeled/all_text.txt,and then it occurs the error.
Thank you in advance.

**Comment by yangkang2318** [comment]
Maybe I find the reason that  the size of unlabeled data we downloaded from "ftp.ncbi.nlm.nih.gov/pubmed/baseline" is 19G.According to your paper,should I sample the unlabeled data and then convert it to weak labeled data?
Thanks!

**Comment by HMJiangGatech** [comment]
Yes, I did sampling at the beginning (before convert it to weak labeled data?). It was too large for our Machine.
